### ISSUE TITLE
trouble installing libspotify with homebrew

### ISSUE THREAD

**Issue by jposemescouilles** [hbashton/spotify-ripper#16]
I tried installing spotify-ripper on a mac ; when I type 

brew install homebrew/binary/libspotify

I get the following error : 

Error: No available formula with the name "libspotify" 
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching taps...
Error: No formulae found in taps.

I tried to install it with cask and it didn't work either. After a bit of searching, I found a workaround : 

brew install mopidy/mopidy/libspotify

ps : thanks for reviving spotify-ripper!

**Comment by Kezzsim** [comment]
Did you have to modify anything else to get this version of Libspotify to run properly? I'm currently getting `Reason: image not found` even after making the symbolic link to the libspotify dynamic library.

**Comment by jposemescouilles** [comment]
No. I didn't even try the symlink way, it worked directly with mopidy. To be precise, I had already installed spotify-ripper on my computer six months ago (which means also xcode, brew, etc) and when it wasn't working I only deleted the .spotify-ripper folder. It may be why it works for me.
